clusterloader2/testing/load/config.yaml

```diff
@@ -441,6 +441,7 @@ steps:
     Method: SchedulingThroughput
     Params:
       action: gather
+      enableViolations: true
```

Comment:
FTR: this is no-op when threshold=0 (it is safe to rollout this without guarding everywhere and control rollout with $schedulerThroughputThreshold).